### PR TITLE
Save students skills

### PR DESCRIPTION
**Pull request by orohellec**

### CODE REVIEW

**Review by orohellec — COMMENTED** [review]
WAIT there is a mistake

**Comment by orohellec** [comment]
> WAIT there is a mistake

It is OK now

**Review by Alexisreau97 — APPROVED** [review]

File: app/controllers/students_controller.rb [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+  def show
+    @student_skills = StudentSkill.where(student_id: @student.id)
+  end
+
   def edit; end
 
   def update
     if @student.update(student_params)
+      redirect_to edit_student_path(@student.id)
       flash[:success] = "Modification enregistrée"
```
Comment (Crysicia) [inline app/controllers/students_controller.rb:20]: Redirect should be after the flash

File: app/controllers/students_controller.rb [hunk 2]
```diff
@@ -24,13 +30,33 @@ def destroy
     redirect_to students_path
   end
 
+  def destroy_skill
+    puts @student.id
```
Comment (Crysicia) [inline app/controllers/students_controller.rb]: There shouldn't be debugging puts in production

File: app/controllers/students_controller.rb [hunk 3]
```diff
@@ -24,13 +30,33 @@ def destroy
...
+    StudentSkill.find_by(student_id: @student.id, skill_id: params[:skill_id]).destroy
+    redirect_to edit_student_path(@student.id)
```
Comment (Crysicia) [inline app/controllers/students_controller.rb:36]: Redirect should be after flash

**Review by Crysicia — DISMISSED** [review]

**Pull request merged**